product_variant_configurator/models/product_configurator.py

```diff
@@ -52,6 +57,19 @@ def _compute_can_be_created(self):
             )
             rec.price_extra = sum(rec.mapped("product_attribute_ids.price_extra"))
 
+    @api.depends("product_tmpl_id", "product_attribute_ids")
+    def _compute_product_id_configurator_domain(self):
+        product_obj = self.env["product.product"]
+        for rec in self:
+            if not rec.product_tmpl_id._origin:
+                # no product template: allow any product
+                rec.product_id_configurator_domain = []
+            else:
+                domain, cont = product_obj._build_attributes_domain(
```

Comment:
```suggestion
                domain, _cont = product_obj._build_attributes_domain(
```

Prefix unused variable with `_`